modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Fields.tsx

```diff
@@ -173,5 +181,24 @@ export default function Fields({
 		],
 	};
 
-	return <FrontendDataSet {...dataSetProps} />;
+	return (
+		<>
+			<FrontendDataSet {...dataSetProps} />
+
+			{showAddFieldModal && (
+				<ModalAddObjectField
+					apiURL={apiURL as string}
+					creationLanguageId={
+						creationLanguageId as Liferay.Language.Locale
+					}
+					objectDefinitionExternalReferenceCode={
+						objectDefinitionExternalReferenceCode
+					}
+					objectFieldTypes={objectFieldTypes}
+					objectName=""
```

Comment:
@danvb112 if the `objectName` is not needed at all times, could we make it optional?

Reply:
Make total sense! Good catch @naraandrade! Done!